lib/modules/manager/composer/utils.ts

```diff
@@ -49,6 +49,22 @@ export function getComposerArguments(
   return args;
 }
 
+export function getComposerUpdateArguments(
+  config: UpdateArtifactsConfig,
+  toolConstraint: ToolConstraint,
+): string {
+  let args = getComposerArguments(config, toolConstraint);
+
+  if (
+    is.string(toolConstraint.constraint) &&
+    api.intersects!(toolConstraint.constraint, '^2.7')
```

Comment:
```suggestion
    api.intersects!(toolConstraint.constraint, '>=2.7')
```

it should also work for composer v3+